src/Algorithms/HLBS25/reach_homog.jl

```diff
@@ -0,0 +1,63 @@
+function reach_homog_HLBS25!(F::Vector{ReachSet{N,S}},
+                             Ω0::S,
+                             Φ::MatrixZonotope{N,MN},
+                             NSTEPS::Integer,
+                             δ::N,
+                             max_order::Integer,
+                             taylor_order::Integer,
+                             ::Val{false},
```

Comment:
```suggestion
                             recursive::Val{false},
```
I know that VSCode suggests to remove unused variables, but with so many arguments it is easier to read the code if you know what they stand for.